src/main.ts

```diff
@@ -279,16 +279,15 @@ export async function run(): Promise<void> {
     excludedFiles !== '' ||
     excludedFilesFromSourceFile !== ''
 
-  if (pathsOutput) {
-    const pathsOutputFile = await tempfile('.txt')
-    await fs.writeFile(pathsOutputFile, pathsOutput, {flag: 'w'})
-
-    core.setOutput('paths-output-file', pathsOutputFile)
-    core.saveState('paths-output-file', pathsOutputFile)
-    core.info(`Successfully created paths-output-file: ${pathsOutputFile}`)
-  } else if (hasCustomPatterns) {
-    throw new Error('No paths found using the specified patterns')
+  if (!pathOutput && hasCustomPatterns) {
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'pathOutput' is not defined.